### PR TITLE
Release/4.3.0

### PR DESCRIPTION
**Pull request by aboedo**
### Identity V3:

In this version, we’ve redesigned the way that user identification works. 
Detailed docs about the new system are available [here](https://docs.revenuecat.com/v3.2/docs/user-ids).

#### New methods
- Introduces `logIn`, a new way of identifying users, which also returns whether a new user has been registered in the system. 
`logIn` uses a new backend endpoint. 
- Introduces `logOut`, a replacement for `reset`. 

#### Deprecations / removals
- deprecates `createAlias` in favor of `logIn`
- deprecates `identify` in favor of `logIn`
- deprecates `reset` in favor of `logOut`
- deprecates `allowSharingPlayStoreAccount` in favor of dashboard-side configuration

    https://github.com/RevenueCat/purchases-android/pull/250
    https://github.com/RevenueCat/purchases-android/pull/260
    https://github.com/RevenueCat/purchases-android/pull/252

### Other changes: 
- Fixed CI issues with creating pull requests
    https://github.com/RevenueCat/purchases-android/pull/324
- Re-enable eTags support to minimize unnecessary network traffic
    https://github.com/RevenueCat/purchases-android/pull/337
- When making a multi-line subscription purchase, all product ids are now sent to the backend.
    https://github.com/RevenueCat/purchases-android/pull/335
- Added `@Throws` annotation to `getPackage`, which could throw `NoSuchElementException`, but it wasn't documented.
    https://github.com/RevenueCat/purchases-android/pull/333

### CODE REVIEW

File: purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt [hunk 1]
```diff
@@ -678,6 +678,10 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
      * @param [newAppUserID] The current user id will be aliased to the app user id passed in this parameter
      * @param [listener] An optional listener to listen for successes or errors.
      */
+    @Deprecated(
+        "Use logIn instead",
+        ReplaceWith("Purchases.sharedInstance.logIn(newAppUserID, LogInCallback?)")
+    )
```
Comment (aboedo) [inline purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt:684]: added this deprecation, I'd missed it in #252

File: RELEASING.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Automatic Releasing
 =========
  1. Create a branch release/x.y.z
- 1. Create a CHANGELOG.latest.md with the changes for the current version (to be used by Fastlane for the github release notes)
+ 1. Update CHANGELOG.latest.md with the changes for the current version (to be used by Fastlane for the github release notes)
```
Comment (aboedo) [inline RELEASING.md:4]: just a copy change to clarify, since the file exists and it's versioned

File: Gemfile.lock [hunk 3]
```diff
@@ -2,25 +2,25 @@ GEM
   remote: https://rubygems.org/
   specs:
     CFPropertyList (3.0.3)
-    addressable (2.7.0)
+    addressable (2.8.0)
```
Comment (aboedo) [inline Gemfile.lock:5]: updated fastlane + dependencies

**Review by taquitos — APPROVED** [review]
🚢 🐐 🥂 🐐

**Pull request merged**